src/main/resources/story/walmart/AddProductToCart.story

```diff
@@ -0,0 +1,24 @@
+Description: Story to test adding product to the cart;
+
+Scenario:  Search for product
+Given I am on a page with the URL 'https://www.walmart.com/'
```

Comment:
In general, once you are clear what will be your main application endpoint, you can introduce a property 
'web-application.main-page-url' in .properties file and specify your main endpoint and then use step 'Given I am on the main application page' every time you want to navigate to the page set in the property.
Ex:web-application.main-page-url=https://walmart.com